test/utils.ts

```diff
@@ -22,7 +22,7 @@ test('Will make a type nullable (null | undefined)', t => {
 });
 
 test('Will make a type not nullable', t => {
```

Comment:
I left this test in as sort of a confirmation that the `Nullable` type was reversible.